app/utils/tracking.ts

```diff
@@ -0,0 +1,45 @@
+declare const ga: any;
+declare const fbq: any;
+declare const twttr: any;
+import { Web3Type } from "../actions/constants";
+import config from "../config";
+
+export const trackCommitEvent = (amount: string, web3Type: Web3Type): void => {
```

Comment:
I think that having a general method for tracking like. trackEvent(name, payload) would generally make more sense. Also this would force marketing team to use consistent event naming. What's the rationale for the current state? For now we can merge it but later we need to rethink this approach